fvt/predictor/predictor_test.go

```diff
@@ -112,6 +112,14 @@ var predictorsArray = []FVTPredictor{
 		differentPredictorName:     "xgboost",
 		differentPredictorFilename: "xgboost-predictor.yaml",
 	},
+	{
+		predictorName:              "pytorch-mar",
+		predictorFilename:          "pytorch-mar-predictor.yaml",
+		currentModelPath:           "fvt/pytorch/pytorch-mar",
```

Comment:
@njhill, I believe this currentModelPath file needs to be created on the the FVT minio image `modelmesh-minio-dev-examples`. How should I go about that?